doc/decisions/1_in_discussion/change_tracking.md

```diff
@@ -43,28 +43,43 @@ Change tracking must:
 - not duplicate data for each plugin that wants change tracking
 - work with all allowed sequences of `kdbGet` and `kdbSet` as [per this decision](operation_sequences.md)
 
+We only want to track changes that are done by the user, not changes that are done by plugins.
+I.e. the scope of change tracking is on what happens _outside_ of `kdbGet` and `kdbSet`.
+
 The library `libelektra-core` must be kept minimal.
 
 ## Assumptions
 
 - It is possible to do change tracking with reasonable memory and computation overhead
-- It is possible to design a single change tracking API that is useful
-  for all existing and future plugins
+- It is possible to design a single change tracking API that is useful for all existing and future plugins
+- False positivies are okay
+
+  - this may happend when some keys may have been changed by the user, but have subsequentially been "unchanged" by a transformation plugin
+    Scenario: hypothetical plugin that converts all values to UPPERCASE.
+    - user gets key `system:/background` with value `RED`
+    - user changes it to `red`
+    - changetracking detects that value has been changed, because `RED` != `red`
+    - plugin changes value to UPPERCASE `RED`
+    - consumers of the changetracking API will now get a false positive if they query whether `system:/background` has been changed
```

Comment:
Let's use something that really exists

```suggestion
    Scenario: `type` plugin configured for boolean conversion with `false`/`true`
    - `system:/background` is stored with value `false`
    - user gets key `system:/background` with value `0` (after conversion by `type`)
    - user changes it to `false`
    - changetracking detects that value has been changed, because `false` != `0`
    - `type` plugin changes value to `0`
    - consumers of the changetracking API will now get a false positive if they query whether `system:/background` has been changed
```

Reply:
Funny, in #4621 it was suggested that we don't name a plugin specifically 😅

Reply:
Hm, I guess there are pros and cons. I just think giving it a name makes this example more readable. I wouldn't even have to be a specific existing plugin. Also I think the `false` -> `0` conversion would be much more common and therefore it would be easier to understand why this is useful/important.

So maybe we go with:

```suggestion
    Scenario: plugin that converts `false`<->`0` and `true`<->`1`
    - `system:/background` is stored with value `false`
    - user gets key `system:/background` with value `0` (after conversion by plugin)
    - user changes it to `false`
    - changetracking detects that value has been changed, because `false` != `0`
    - plugin changes value from `false` to `0`
    - consumers of the changetracking API will now get a false positive if they query whether `system:/background` has been changed
```

Note also the additional line at the beginning explaining how the value is stored.